<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="sukeerthsahana" failures="5" tests="29" name="dsAlgoTests.QueueTest" time="120.964" errors="0" timestamp="2025-02-16T15:06:06 CST" skipped="5">
  <testcase classname="dsAlgoTests.QueueTest" name="navigatetoImplementationOfQueueInPython" time="3.635"/>
  <system-out/>
  <testcase classname="dsAlgoTests.QueueTest" name="navigatetoImplementationOfQueueInPythonTryEditor" time="4.079"/>
  <system-out/>
  <testcase classname="dsAlgoTests.QueueTest" name="navigatetoImplementationUsingCollectionsDeque" time="3.959"/>
  <system-out/>
  <testcase classname="dsAlgoTests.QueueTest" name="navigatetoImplementationUsingCollectionsDequeTryEditor" time="4.093"/>
  <system-out/>
  <testcase classname="dsAlgoTests.QueueTest" name="navigateToImplementationUsingArray" time="3.521"/>
  <system-out/>
  <testcase classname="dsAlgoTests.QueueTest" name="navigateToImplementationUsingArrayTryEditor" time="4.165"/>
  <system-out/>
  <testcase classname="dsAlgoTests.QueueTest" name="navigateToQueueOperations" time="3.864"/>
  <system-out/>
  <testcase classname="dsAlgoTests.QueueTest" name="navigateToQueueOperationsTryEditor" time="4.290"/>
  <system-out/>
  <testcase classname="dsAlgoTests.QueueTest" name="navigateToPracticeQuestionOfImplementationOfQueueInPython" time="3.382"/>
  <system-out/>
  <testcase classname="dsAlgoTests.QueueTest" name="listOfPracticeQuestionOfImplementationOfQueueInPython" time="4.023">
    <skipped/>
  </testcase> <!-- listOfPracticeQuestionOfImplementationOfQueueInPython -->
  <system-out/>
  <testcase classname="dsAlgoTests.QueueTest" name="listOfPracticeQuestionOfImplementationOfQueueInPython" time="3.977">
    <failure message="No list of practice question found" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: No list of practice question found
at org.testng.Assert.fail(Assert.java:99)
at dsAlgoTests.QueueTest.listOfPracticeQuestionOfImplementationOfQueueInPython(QueueTest.java:206)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:418)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
]]>
    </failure>
  </testcase> <!-- listOfPracticeQuestionOfImplementationOfQueueInPython -->
  <system-out/>
  <testcase classname="dsAlgoTests.QueueTest" name="navigateToPracticeQuestionImplementationUsingCollectionsDeque" time="3.446"/>
  <system-out/>
  <testcase classname="dsAlgoTests.QueueTest" name="listOfPracticeQuestionImplementationUsingCollectionsDeque" time="3.649"/>
  <system-out/>
  <testcase classname="dsAlgoTests.QueueTest" name="checkErrorMeaasgeWhenExecutingWithoutCodeInImplementationOfQueueInPython" time="4.226">
    <skipped/>
  </testcase> <!-- checkErrorMeaasgeWhenExecutingWithoutCodeInImplementationOfQueueInPython -->
  <system-out/>
  <testcase classname="dsAlgoTests.QueueTest" name="checkErrorMeaasgeWhenExecutingWithoutCodeInImplementationOfQueueInPython" time="4.905">
    <failure message="Test failed: Expected an error message when clicking the &#039;Run&#039; button without entering code, but no message was displayed" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Test failed: Expected an error message when clicking the 'Run' button without entering code, but no message was displayed
at org.testng.Assert.fail(Assert.java:99)
at dsAlgoTests.QueueTest.checkErrorMeaasgeWhenExecutingWithoutCodeInImplementationOfQueueInPython(QueueTest.java:257)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:418)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
]]>
    </failure>
  </testcase> <!-- checkErrorMeaasgeWhenExecutingWithoutCodeInImplementationOfQueueInPython -->
  <system-out/>
  <testcase classname="dsAlgoTests.QueueTest" name="checkErrorMessageWhenExecutingWithoutCodeInImplementationUsingCollectionsDeque" time="4.527">
    <failure message="Test failed: Expected an error message when clicking the &#039;Run&#039; button without entering code, but no message was displayed" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Test failed: Expected an error message when clicking the 'Run' button without entering code, but no message was displayed
at org.testng.Assert.fail(Assert.java:99)
at dsAlgoTests.QueueTest.checkErrorMessageWhenExecutingWithoutCodeInImplementationUsingCollectionsDeque(QueueTest.java:277)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:418)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
]]>
    </failure>
  </testcase> <!-- checkErrorMessageWhenExecutingWithoutCodeInImplementationUsingCollectionsDeque -->
  <system-out/>
  <testcase classname="dsAlgoTests.QueueTest" name="checkErrorMessageWhenExecutingWithoutCodeInImplementationUsingCollectionsDeque" time="3.956">
    <skipped/>
  </testcase> <!-- checkErrorMessageWhenExecutingWithoutCodeInImplementationUsingCollectionsDeque -->
  <system-out/>
  <testcase classname="dsAlgoTests.QueueTest" name="checkErrorMeaasgeWhenExecutingWithoutCodeInImplementationUsingArray" time="4.059">
    <failure message="Test failed: Expected an error message when clicking the &#039;Run&#039; button without entering code, but no message was displayed" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Test failed: Expected an error message when clicking the 'Run' button without entering code, but no message was displayed
at org.testng.Assert.fail(Assert.java:99)
at dsAlgoTests.QueueTest.checkErrorMeaasgeWhenExecutingWithoutCodeInImplementationUsingArray(QueueTest.java:296)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:418)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
]]>
    </failure>
  </testcase> <!-- checkErrorMeaasgeWhenExecutingWithoutCodeInImplementationUsingArray -->
  <system-out/>
  <testcase classname="dsAlgoTests.QueueTest" name="checkErrorMeaasgeWhenExecutingWithoutCodeInImplementationUsingArray" time="4.220">
    <skipped/>
  </testcase> <!-- checkErrorMeaasgeWhenExecutingWithoutCodeInImplementationUsingArray -->
  <system-out/>
  <testcase classname="dsAlgoTests.QueueTest" name="checkErrorMeaasgeWhenExecutingWithoutCodeInQueueOperations" time="4.653">
    <skipped/>
  </testcase> <!-- checkErrorMeaasgeWhenExecutingWithoutCodeInQueueOperations -->
  <system-out/>
  <testcase classname="dsAlgoTests.QueueTest" name="checkErrorMeaasgeWhenExecutingWithoutCodeInQueueOperations" time="4.165">
    <failure message="Test failed: Expected an error message when clicking the &#039;Run&#039; button without entering code, but no message was displayed" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Test failed: Expected an error message when clicking the 'Run' button without entering code, but no message was displayed
at org.testng.Assert.fail(Assert.java:99)
at dsAlgoTests.QueueTest.checkErrorMeaasgeWhenExecutingWithoutCodeInQueueOperations(QueueTest.java:315)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:418)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
]]>
    </failure>
  </testcase> <!-- checkErrorMeaasgeWhenExecutingWithoutCodeInQueueOperations -->
  <system-out/>
  <testcase classname="dsAlgoTests.QueueTest" name="verifyCodeExecutionForQueueTryEditor" time="4.489"/>
  <system-out/>
  <testcase classname="dsAlgoTests.QueueTest" name="verifyCodeExecutionForQueueTryEditor" time="4.409"/>
  <system-out/>
  <testcase classname="dsAlgoTests.QueueTest" name="verifyCodeExecutionForQueueTryEditor" time="4.578"/>
  <system-out/>
  <testcase classname="dsAlgoTests.QueueTest" name="verifyCodeExecutionForQueueTryEditor" time="4.962"/>
  <system-out/>
  <testcase classname="dsAlgoTests.QueueTest" name="verifyCodeExecutionForQueueTryEditor" time="4.406"/>
  <system-out/>
  <testcase classname="dsAlgoTests.QueueTest" name="verifyCodeExecutionForQueueTryEditor" time="4.368"/>
  <system-out/>
  <testcase classname="dsAlgoTests.QueueTest" name="verifyCodeExecutionForQueueTryEditor" time="4.537"/>
  <system-out/>
  <testcase classname="dsAlgoTests.QueueTest" name="verifyCodeExecutionForQueueTryEditor" time="4.421"/>
  <system-out/>
</testsuite> <!-- dsAlgoTests.QueueTest -->
